src/Idris/Parser/Expr.hs

```diff
@@ -678,16 +679,17 @@ app syn = do f <- simpleExpr syn
                                f
                                (PMatchApp fc ff)
                                (PRef fc [] (sMN 0 "match")))
-              <?> "matching application expression") <|> (do
-             fc <- getFC
-             i <- get
-             args <- many (do notEndApp; arg syn)
-             wargs <- if withAppAllowed syn && not (inPattern syn)
-                         then many (do notEndApp; reservedOp "|"; expr' syn)
-                         else return []
-             case args of
-               [] -> return f
-               _  -> return (withApp fc (flattenFromInt fc f args) wargs))
+                   <?> "matching application expression") <|>
+               (do
+                   fc <- getFC
+                   i <- get
+                   args <- many (do notEndApp; arg syn)
+                   wargs <- if withAppAllowed syn && not (inPattern syn)
+                            then many (do notEndApp; reservedOp "|"; expr' syn)
```

Comment:
Shouldn't `then` and `else` be two spaces in under the `if`? Is this a `stylish-haskell` config option, or does it ignore this?